lib/puppet_x/vmware/vmware_lib/puppet_x/puppetlabs/transport/vsphere.rb

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
In relation to my other comment about the vsphere transport no longer being in vmware_lib, we should probably just delete this one as to not cause confusion